frame/bonded-finance/src/lib.rs

```diff
@@ -204,10 +212,14 @@ pub mod pallet {
 		/// appropriate funds.
 		///
 		/// Emits a `NewOffer`.
-		#[pallet::weight(10_000)]
-		pub fn offer(origin: OriginFor<T>, offer: BondOfferOf<T>) -> DispatchResult {
+		#[pallet::weight(T::WeightInfo::offer())]
```

Comment:
if API generator generates doc too, than better to document what is keep_alive :)